### ISSUE TITLE
Loss converged upon unfreezing aggregator but results bad

### ISSUE THREAD

**Issue by aman190202** [facebookresearch/vggt#307]
Hi, I recently finetuned VGGT's camera head on COLMAP data ; While unfreezing the camera head resulted in convergence, upon inferencing, the results are worse than the pretrained VGGT's output. 

What are some things I should keep an eye on?

**Comment by shiyao-li** [comment]
Hi, I’m facing the same issue. Have you found a solution?

**Comment by jytime** [comment]
Hi can you try to use a smaller lr, e.g., 10% of the original one?

**Comment by surpoloyang** [comment]
Hello, can you share how to fine-tune VGGT using COLMAP data? :)

> Hi, I recently finetuned VGGT's camera head on COLMAP data ; While unfreezing the camera head resulted in convergence, upon inferencing, the results are worse than the pretrained VGGT's output.嗨，我最近在 COLMAP 数据上微调了 VGGT 的相机头；在解冻相机头后虽然实现了收敛，但在推理时，结果比预训练的 VGGT 输出更差。
> 
> What are some things I should keep an eye on?我应该关注哪些事项？
